test/operator/test_operator.py

```diff
@@ -120,15 +120,14 @@ def test_produce_elements_in_hilbert(op, attr):
     "op", [pytest.param(op, id=name) for name, op in operators.items()]
 )
 def test_is_hermitean(op):
-    rng = nk.jax.PRNGSeq(0)
+    rng = nk.jax.PRNGSeq(20)
```

Comment:
Why 20? (I guess "why not" would be a legitimate answer...)

Reply:
i just wanted a diffrent one from the other test set...

Reply:
beautiful, hopefully it doesn't break for seed 21

Reply:
try and find out...